third_party/toolchains/BUILD

```diff
@@ -29,21 +29,11 @@ platform(
         "@bazel_tools//platforms:windows",
         "@bazel_tools//tools/cpp:msvc",
     ],
-    remote_execution_properties = """
-        properties: {
-          name: "container-image"
-          value:"docker://gcr.io/grpc-testing/rbe_windows_toolchain@sha256:75728e7d6d804090f71095e5fec627b18cfa1f47adc52096c209f2a687e06b2e"
-        }
-        properties: {
-          name: "gceMachineType"  # Small machines for majority of tests.
-          value: "n1-highmem-2"
-        }
-        properties:{
-            name: "OSFamily"
-            value: "Windows"
-        }
-
-        """,
+    exec_properties = {
+        "container-image" : "docker://gcr.io/grpc-testing/rbe_windows_toolchain@sha256:75728e7d6d804090f71095e5fec627b18cfa1f47adc52096c209f2a687e06b2e",
+        "gceMachineType" : "n1-highmem-2",
```

Comment:
nit: keep comment `# Small machines for majority of tests`

Reply:
right now all the windows tests are actually running with small machine, so the missing comment is accurate in a sense.